lib/screens/detail/survey_detail_screen.dart

```diff
@@ -94,18 +96,28 @@ class SurveyDetailScreenState extends State<SurveyDetailScreen> {
 
   Widget _getQuestionContentChild(SurveyQuestionModel question) {
     return Consumer(builder: (_, widgetRef, child) {
+      final viewModel = widgetRef.read(surveyDetailViewModelProvider.notifier);
       switch (question.displayType) {
         case DisplayType.choice:
           return MultipleChoiceAnswers(
             answers: question.answers,
             onChoiceClick: (answerId) {
-              widgetRef
-                  .read(surveyDetailViewModelProvider.notifier)
-                  .updateChoiceAnswer(
-                    questionId: question.id,
-                    answerId: answerId,
-                    pickType: question.pick,
-                  );
+              viewModel.updateChoiceAnswer(
+                questionId: question.id,
+                answerId: answerId,
+                pickType: question.pick,
+              );
```

Comment:
Instead, calling `widgetRef.read` here
```suggestion
              widgetRef.read(surveyDetailViewModelProvider.notifier).updateChoiceAnswer(
                questionId: question.id,
                answerId: answerId,
                pickType: question.pick,
              );
```